libmscore/page.cpp

```diff
@@ -273,7 +273,7 @@ void Page::doRebuildBspTree()
             r = abbox();
 
       bspTree.initialize(r, n);
-      scanElements(&bspTree, &bspInsert, false);
+      scanElements(&bspTree, &bspInsert, true);
```

Comment:
Why this change? It is the reason for [Hidden Staves appear when Hide Empty Staves enabled](https://musescore.org/en/node/276537).

Reply:
If we set `true` here, hidden elements will be taken into account when building the tree which is not expected. It makes the mentioned issue even harder to fix.